package.json

```diff
@@ -49,7 +49,7 @@
     "get-stdin": "^5.0.1",
     "glob": "^7.1.2",
     "is-plain-obj": "^1.1.0",
-    "is-url": "^1.2.2",
+    "is-url": "1.2.2",
```

Comment:
Ok yeah. That might work. But really, we should be using the latest version of `is-url`. Looking at their [changelog](https://github.com/segmentio/is-url/commits/master), bumping to version 1.2.4 should fix everything.